commands-yml/commands/session/back.yml

```diff
@@ -0,0 +1,70 @@
+---
+name: Go Back
+short_description: Navigate backwards in the browser history, if possible (Web context only)
+
+example_usage:
+  java:
+    |
+      driver.back();
```

Comment:
@dpgraham AFAIK, i'm not aware of back method in iOS or AndroidDriver in java client. Would review the entire PR once before we get this in. So hold on for some moment.

Reply:
@SrinivasanTarget any update on this?